### PR TITLE
Sports Betting - My Accounts - My Favorites

### PR DESCRIPTION
**Pull request by rubyweb3**
https://github.com/AugurProject/augur/issues/6974

Screen Shot 2020-04-15 at 6 17 43 PM
Screen Shot 2020-04-15 at 6 17 34 PM

### CODE REVIEW

File: packages/augur-ui/src/modules/portfolio/components/favorites/favorites.styles.less [hunk 1]
```diff
@@ -57,4 +57,29 @@
...
+        }
+      }
+    }
+
+    > div:last-of-type {
+      min-height: @size-414;
+    }
+  }
+
+  .Watchlist__EmptyDisplay {
```
Comment (rubyweb3) [inline packages/augur-ui/src/modules/portfolio/components/favorites/favorites.styles.less]: I find that the height of Favorites component will be different when it's empty and not empty. I have spent some time on the current implementation. What do you think about it?

Comment (JohnDanz) [reply packages/augur-ui/src/modules/portfolio/components/favorites/favorites.styles.less]: It looks fine to me however can we change the class name? we spent time to remove classes with `__` or `-` in their names. Maybe just call this `EmptyDisplay` or `WatchlistEmptyDisplay`

**Review by JohnDanz — CHANGES_REQUESTED** [review]
minor request

File: packages/augur-ui/src/modules/portfolio/components/favorites/favorites.styles.less (hunk 1 above)
Comment (rubyweb3) [reply packages/augur-ui/src/modules/portfolio/components/favorites/favorites.styles.less]: Thanks. Updated.

**Review by JohnDanz — APPROVED** [review]

**Pull request merged**